python_modules/dagster/dagster/_core/storage/event_log/base.py

```diff
@@ -292,6 +304,20 @@ def get_latest_materialization_events(
     ) -> Mapping[AssetKey, Optional[EventLogEntry]]:
         pass
 
+    @abstractmethod
+    def get_event_tags_for_asset(
+        self, asset_key: AssetKey, tag_key: Optional[str] = None, tag_value: Optional[str] = None
```

Comment:
Instead of separate tag_key/tag_value arguments, maybe this should be `filter_tags` that takes a mapping?  If we ever want to support more than 2 dimensions.